### DISCUSSION TITLE
Save to Folder menu option greyed out

### DISCUSSION THREAD

**Discussion by danxshap** [TabularEditor/TabularEditor3#646] [🙋♀️TE3 Q&A] [answered]
Hi, I just purchased a license to Tabular Editor 3 and I'm trying to figure out how to use the [Save to Folder feature](https://docs.tabulareditor.com/onboarding/parallel-development.html#configuring-save-to-folder).

The problem I'm running into is that the menu option is greyed out:

![](https://d.pr/i/PIagAC+)

This is all I'm doing:

1. Open a .pbix file containing my dataset
2. Open TE3 (v3.3.4) via the External Tools ribbon
3. Open the File menu in TE3.

When I do it with TE2 (v2.16.6) with the exact same .pbix file, the menu option is enabled and I'm able to save to folder.

Am I doing something wrong?

**Comment by otykier** [comment, marked as answer]
Hi Daniel. Which edition did you purchase? Saving and loading model metadata to/from files requires Business or Enterprise Edition. Desktop Edition is only intended as an external tool for Power BI Desktop: https://docs.tabulareditor.com/te3/editions.html

**Reply by otykier** [reply]
Also, don’t hesitate to contact sales@tabulareditor.com if you have more questions about the license, or if you’d like to upgrade or get a refund for your current subscription.

**Reply by danxshap** [reply]
Ah, I got the Desktop Edition, that explains it. Thanks for the super quick reply, by the way!

I guess I was confused because the save/export is a (free) feature in TE2, so it didn't even cross my mind that it'd be a premium-only feature in TE3.

To be clear, I'm not looking to _load_ model metadata — I know that's of no use to me since I'm just using Power BI Desktop and I can't update the .pbix file with updated model metadata.

But I'd like to save the model data and commit it to git so I can have a nice diff history of what has changed and when (even if I can't revert to a previous version without manually applying changes in PBI Desktop).

Would your recommendation be for me to just use TE2 for this purpose, and TE3 for everything else?

**Reply by otykier** [reply]
Yes, TE2 would be a good fit in that case. You could even create a simple external tool for PBI Desktop that uses TE2’s command line to save the pbix as a folder and commit to git (you can do that using PowerShell or a .cmd file). Thus, you would only need to click a single button in the external tools ribbon whenever you wanted to store a snapshot of the model in git. See here for more information on creating custom external tools: https://youtu.be/Kfwr3UlO5O0

**Reply by danxshap** [reply]
Awesome, that's very helpful. Thank you!
